setup.cfg

```diff
@@ -15,3 +15,6 @@ universal=1
 max-line-length = 120
 exclude =
     satpy/readers/li_l2.py
+    satpy/readers/scatsat1_l2b.py
```

Comment:
Why?

Reply:
This is an old reader that hasn't been updated yet and has a lot of style issues (same for li_l2.py and the others).

Reply:
To clarify more, these modules use the deprecated Dataset import. To fully fix the styling in them I would have to update them to use xarray as flake8 (with all the various plugins we use) will likely complain about things in some way.